BraintreeCore/src/main/java/com/braintreepayments/api/BTAPITiming.kt

```diff
@@ -0,0 +1,5 @@
+package com.braintreepayments.api
+
+fun interface BTAPITiming {
```

Comment:
I think we can remove `BT` from these names, that's an iOS specific thing for Obj-C interoperability.

Reply:
Thanks for your feedback. Addressed: [8bb8aae](https://github.com/braintree/braintree_android/pull/1032/commits/8bb8aae8ee7df8da642079f1ecb6c6f1e05f067b)